shopify-theme-api/index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+const Shopify = require('shopify-api-node');
+const core = require('@actions/core');
+const shopifyStore = process.env.SHOPIFY_STORE;
+const apiKey = process.env.SHOPIFY_API_KEY;
+const apiSecret = process.env.SHOPIFY_API_SECRET;
+const themeName = process.env.SHOPIFY_THEME_NAME;
+
+const shopify = new Shopify({
+  shopName: shopifyStore,
+  apiKey: apiKey,
+  password: apiSecret
+});
+
+async function run(action) {
+  try {
+    let themes = await shopify.theme.list()
+    themes.forEach(theme => {
+      if(action == 'get' || action == '') {
+        console.log(theme)
+      }
+      else if (theme.name == themeName) {
```

Comment:
```suggestion
      else if (theme.name === themeName) {
```

While we're not enforcing JS rules with `eslint` in this repo at the moment, it's best practice to use the strict equality operator (`===`) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators